api/src/main/java/org/pmiops/workbench/cohortreview/util/Operator.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.cohortreview.util;
+
+public enum Operator {
+
+    EQUALS("=");
```

Comment:
Are you planning on supporting other operators in future? How does this get used?

Reply:
We will have other operators in the future, but we still don't have a decision on what others yet. Will probably be >, <, >=, <=, like... etc. These will be used with the filters.